tests/test_compile.py

```diff
@@ -1,9 +1,9 @@
-from . import c_eval
-from . import c_exec
-from . import e_eval
 from RestrictedPython import compile_restricted
 from RestrictedPython import CompileResult
 from RestrictedPython._compat import IS_PY2
+from tests import c_eval
+from tests import c_exec
+from tests import e_eval
```

Comment:
@loechel @stephan-hof I put these functions into `__init__.py`  because they are temporary. When we delete the old implementation we will delete them, too. (As I understand we will delete the old implementation before merging the `Python3` branch to master.)